Resolve TODO's (#2)
djuszczakk/tizen-docs: docs/application/web/guides/connectivity/bluetooth.md

You can use Bluetooth functionalities in your application, such as managing the local Bluetooth adapter, bonding, and exchanging data between Bluetooth-enabled devices. The Bluetooth standard provides a peer-to-peer (P2P) data exchange functionality over short distance between compliant devices.

> <!-- TODO #00: features are for application filtering, not related with privileges and not mentioned at most of guides

Reviewer: If features won't be mentioned, I think the comment can be removed.

Author: All TODO comments will be removed in the last commit.

	   - write about uuidsEqual()
 -->
## Handling UUIDs and binary data in Bluetooth API

> ### Hnadling UUIDs

Reviewer 2: Hnadling -> Handling

   For example: "e72ad71b".
   * 16-bit representation: "D<sub>1</sub>D<sub>2</sub>D<sub>3</sub>D<sub>4</sub>", equivalent to "0000D<sub>1</sub>D<sub>2</sub>D<sub>3</sub>D<sub>4</sub>-0000-1000-8000-00805F9B34FB", where D<sub>1</sub>..D<sub>4</sub> stand for hexadecimal digits.  
   For example: "d182".

Unless stated otherwise, each function taking a UUID as a parameter will accept any of the 3 forms and make appropriate conversions.  
Unless stated otherwise, each UUID returned from a function and each UUID attribute of an object may be a 16-bit, 32-bit or 128-bit UUID.
Bluetooth API functions are case-insensitive with regard to UUIDs - lowercase (abcdef) and uppercase (ABCDEF) characters are recognized as valid hexadecimal digits.

The 128-bit UUID that is the base for UUIDs having 16-bit or 32-bit equivalents is defined available through the API in `BluetoothManager` as `BASE_UUID`.

### Handling binary data
The `Bytes` type, that aggregates all types in Bluetooth API used to pass binary data. It can be either a *byte[]* or a *DOMString* or a *Uint8Array*.

`BluetoothManager` methods to manipulate binary data:

>    * toByteArray()

Reviewer 2: I think that all function names outside code examples should be enclosed with `back quote`.

      ```
      var dataInt8Array = new Int8Array([24, 177]);
      var dataUint8Array = new Uint8Array([24, 177]);
      var dataString = "0x18b1";

      var first = tizen.bluetooth.toByteArray(dataInt8Array);
      var second = tizen.bluetooth.toByteArray(dataUint8Array);
      var third = tizen.bluetooth.toByteArray(dataString);
      ```

>       Both first, second and third variables are equal arrays.

Reviewer: According to https://dictionary.cambridge.org/dictionary/english/both, "both" should be used when referring to 2 things.
```suggestion
      `first`, `second` and `third` variables are equal arrays.
```

   * `toDOMString()`
      ```
      var dataInt8Array = new Int8Array([24, 177]);
      var dataUint8Array = new Uint8Array([24, 177]);
      var dataString = "0x18b1";

      var first = tizen.bluetooth.toDOMString(dataInt8Array);
      var second = tizen.bluetooth.toDOMString(dataUint8Array);
      var third = tizen.bluetooth.toDOMString(dataString);
      ```

>       Both first, second and third variables are equal to *"0x18b1"*.

Reviewer: According to https://dictionary.cambridge.org/dictionary/english/both, "both" should be used when referring to 2 things.
```suggestion
      `first`, `second` and `third` variables are equal arrays.
```

   * `toUint8Array()`
      ```
      var dataInt8Array = new Int8Array([24, 177]);
      var dataUint8Array = new Uint8Array([24, 177]);
      var dataString = "0x18b1";

      var first = tizen.bluetooth.toUint8Array(dataInt8Array);
      var second = tizen.bluetooth.toUint8Array(dataUint8Array);
      var third = tizen.bluetooth.toUint8Array(dataString);
      ```

>       Both first, second and third variables are equal.

Reviewer: According to https://dictionary.cambridge.org/dictionary/english/both, "both" should be used when referring to 2 things.
```suggestion
      `first`, `second` and `third` variables are equal.
```

>    * `uuidTo128bit()`

Reviewer: This should go to the section about UUIDs.

Author: Done.

      ```
      var uuidFrom16bits = tizen.bluetooth.uuidTo128bit("1234");
      var uuidFrom32bits = tizen.bluetooth.uuidTo128bit("ab5690ef");
      var uuidFrom128bits = tizen.bluetooth.uuidTo128bit("abcdef01-2345-6789-abcd-ef0123456789");
      ```

      uuidFrom16bits is equal to "00001234-0000-1000-8000-00805f9b34fb"  
      uuidFrom32bits is equal to "ab5690ef-0000-1000-8000-00805f9b34fb"  
      uuidFrom128bits is equal to "abcdef01-2345-6789-abcd-ef0123456789"

>    * `uuidToShortestPossible()`

Reviewer: This should go to the section about UUIDs.

Author: Done.

      ```
      var from16Bit = tizen.bluetooth.uuidToShortestPossible("1234");
      var from32Bit = tizen.bluetooth.uuidToShortestPossible("0000acdf");
      var from128BitFirst = tizen.bluetooth.uuidToShortestPossible("ab5690ef-0000-1000-8000-00805F9B34FB");
      var from128BitSecond = tizen.bluetooth.uuidToShortestPossible("abcdef01-2345-6789-abcd-ef0123456789");
      ```

      from16Bit is equal to "1234"  
      from32Bit is equal to "acdf"  
      from128BitFirst is equal to "ab5690ef"  
      from128BitSecond is equal to "abcdef01-2345-6789-abcd-ef0123456789"

>    * `uuidsEqual()`

Reviewer: This should go to the section about UUIDs.

Author: Done.

      ```
      var first = tizen.bluetooth.uuidsEqual("1234", "00001234");
      var second = tizen.bluetooth.uuidsEqual("ab5690ef", "ab5690ef-0000-1000-8000-00805F9B34FB");
      var third = tizen.bluetooth.uuidsEqual("abcdef01-2345-6789-abcd-ef0123456789", "abcdef01");
      ```

>       Both first and second variables are true. Variable third is false.

Reviewer: ```suggestion
      Both `first` and `second` are `true`. `third` is false.
```

> <!-- END #01 -->

Reviewer: This TODO comment can be removed.

Author: All TODO comments will be removed in the last commit.

> <!-- TODO #03:

Reviewer: This TODO comment can be removed.

Author: All TODO comments will be removed in the last commit.

   }
   ```

3. Define a callback for the scan operation that connects to a not connected found device and stops the scan.

>    Within the callback request, check if found device is connected and establish a connection with the not connected device using the `isConnected()` and `connect()` methods of the `BluetoothLEDevice` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothLEDevice), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothLEDevice) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothLEDevice) applications):

Reviewer: ```suggestion
   Within the callback request, check if the found device is connected and establish a connection with the not connected device using the `isConnected()` and `connect()` methods of the `BluetoothLEDevice` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothLEDevice), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothLEDevice) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothLEDevice) applications):
```
